src/tasks.h

```diff
@@ -81,7 +81,7 @@ class Dispatcher : public ThreadHolder<Dispatcher> {
 		std::mutex taskLock;
 		std::condition_variable taskSignal;
 
-		std::list<Task*> taskList;
+		std::vector<Task*> taskList;
```

Comment:
Maybe we could use taskList.reserve in Dispatcher constructor?

Reply:
We could, i did not bothered about reserving in both taskList and tmpTaskList since(atleast in visual studio) the vector keeps the capacity after copying another vector and call to clear(so its means if we add 1500 tasks to taskList in one time like in server startup it will keep a 1800~ capacity for taskList and 1500 for tmpTaskList).